src/main/java/racing/RacingCarList.java

```diff
@@ -0,0 +1,64 @@
+package racing;
+
+import racing.strategy.MoveStrategy;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+public class RacingCarList {
```

Comment:
일급 컬렉션을 사용할 땐 다음과 같이 복수명사를 이요해주세요!
```suggestion
public class RacingCars {
```